grafana-plugin/src/pages/incident/parts/PagedUsers.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useCallback } from 'react';
+
+import { HorizontalGroup, IconButton } from '@grafana/ui';
+import cn from 'classnames/bind';
+
+import Avatar from 'components/Avatar/Avatar';
+import Text from 'components/Text/Text';
+import WithConfirm from 'components/WithConfirm/WithConfirm';
+import { WithPermissionControl } from 'containers/WithPermissionControl/WithPermissionControl';
+import { Alert } from 'models/alertgroup/alertgroup.types';
+import { User } from 'models/user/user.types';
+import { UserActions } from 'utils/authorization';
+
+import styles from './../Incident.module.scss';
```

Comment:
This can be an absolute path instead.

Reply:
it's ok in this case because we import `scss` file from `parts` subfolder